node/tessel-export.js

```diff
@@ -85,6 +94,73 @@ Tessel.prototype.close = function() {
   }, this);
 };
 
+Tessel.prototype.pwmFrequency = function(frequency, cb) {
+  if (frequency < Tessel.pwmMinFrequency || frequency > Tessel.pwmMaxFrequency) {
+    throw new Error(`${frequency} is outside the bounds of ${Tessel.pwmMinFrequency}-${Tessel.pwmMaxFrequency} Hz`);
```

Comment:
The almost-convention used elsewhere in this file is: "{operation} value must be between LOW and HIGH"